metrics/firebase/functions/lib/main.dart

```diff
@@ -42,7 +47,78 @@ Future<void> onBuildAddedHandler(DocumentSnapshot snapshot, _) async {
     statusFields: [statusFieldIncrement],
   );
 
-  await _updateBuildDay(snapshot, buildDayData);
+  final dataJson = Map<String, dynamic>.from(buildData.toJson());
+  dataJson.addAll({'id': buildId});
```

Comment:
Would it be great to have some `BuildDataSerializer` class that will be responsible for serializing the `BuildData` into a JSON-encodable map? It will allow us to create a `BuildData` instance with a non-null `id` field and then serialize it using this serializer. What do you think?